nana/modules/meme.py

```diff
@@ -279,4 +281,37 @@ async def fake_ss(client, message):
                     random_id=client.rnd_id(),
                 )
             )
-        )
\ No newline at end of file
+        )
+
+
+@app.on_message(filters.user(AdminSettings) & filters.command("g", Command))
+async def glitch(client, message):
+    cmd = message.command
+    amount = ""
+    if len(cmd) > 1:
+        amount = " ".join(cmd[1:])
+    elif len(cmd) == 1:
+        amount = '2'
+    profile_photo = "nana/downloads/pfp.jpg"
+    glitched_gif = "nana/downloads/glitched_pfp.gif"
+    replied = message.reply_to_message
+    if not replied:
+        await message.delete()
+        return
+    user = await client.get_users(replied.from_user.id)
+    await client.download_media(user.photo.big_file_id, file_name=profile_photo)
```

Comment:
Codacy found an issue: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)